src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackCcdApi.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import feign.FeignException;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+import static java.lang.String.format;
+
+@Component
+public class CallbackCcdApi {
+    private final CoreCaseDataApi ccdApi;
+
+    public CallbackCcdApi(CoreCaseDataApi ccdApi) {
+        this.ccdApi = ccdApi;
+    }
+
+    private CaseDetails retrieveCase(String caseRef, CcdAuthenticator authenticator) {
+        return ccdApi.getCase(authenticator.getUserToken(), authenticator.getServiceToken(), caseRef);
+    }
+
+    private StartEventResponse startAttachScannedDocs(String caseRef,
+                                                      CcdAuthenticator authenticator,
+                                                      String jurisdiction,
+                                                      String caseTypeId) {
```

Comment:
we normally format it like this:
```
private StartEventResponse startAttachScannedDocs(
    String caseRef,
    CcdAuthenticator authenticator,
    String jurisdiction,
    String caseTypeId
) {
```
(less spacing changes when method name (or return type name) changes.

OK to leave it like that, but would be worth keeping it consistent in the future.